paperqa/docs.py

```diff
@@ -205,7 +205,15 @@ def set_client(
         embedding_client: Any | None = None,
     ):
         if client is None and isinstance(self.llm_model, OpenAILLMModel):
-            client = AsyncOpenAI()
+            if (api_key := os.environ.get("ANYSCALE_API_KEY")) and (
```

Comment:
I think LiteLLM does this for us fwiw. So in the future we can consolidate this away

Reply:
for sure